### PR TITLE
Fix Read the Docs rendering

### LINKED ISSUE #19: API docs missing at RTD
**Describe the bug**

Looking at https://openscm-units.readthedocs.io/en/latest/unit_registry.html, it is very empty. )-:

**Expected behavior**

There should be API docs autogenerated from the module.

**Additional context**

If I `make docs` locally, the API docs are there, so I have no idea why they are missing at RTD

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

Closes #19 

New read the docs state can be viewed at https://openscm-units.readthedocs.io/en/19-fix-rtd/index.html

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable) (N/A)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <https://github.com/openscm/openscm-units/pull/XX>`_) Added feature which does something
- (`#XX <https://github.com/openscm/openscm-units/pull/XX>`_) Fixed bug identified in (`#YY <https://github.com/openscm/openscm-units/issues/YY>`_)
```

### CODE REVIEW

**Comment by znicholls** [comment]
@mikapfl this should fix it

File: src/openscm_units/data/mixtures.py [hunk 1]
```diff
@@ -407,3 +399,18 @@
...
+key is the name of a component of the mixture and the value is a list in which
+the first element is the standard composition, the second element is the positive
+composition tolerance and the third element is the negative composition tolerance.
+All values are given in mass percentage.
+
+Sources:
+
+- ANSI/ASHRAE Standard 34-2019, p. 9ff, ISSN 1041-2336, https://www.techstreet.com/ashrae/standards/ashrae-15-2019-packaged-w-34-2019?product_id=2046531
+
+- https://en.wikipedia.org/wiki/List_of_refrigerants (for common names), last update: 2020-12-16
```
Comment (mikapfl) [inline src/openscm_units/data/mixtures.py:415]: The "last update" refers to the whole dict, not to the wikipedia source.

**Comment by znicholls** [comment]
Updated

File: docs/source/data.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Data API
+--------
+
+.. automodule:: openscm_units.data
+
+Mixtures API
+============
+
+.. automodule:: openscm_units.data.mixtures
```
Comment (mikapfl) [inline docs/source/data.rst:11]: Using this
```suggestion
.. autodata:: openscm_units.data.mixtures.MIXTURES
   :annotation:
```
the value of MIXTURES is not printed in the docs, which makes them easier to read.

**Review by mikapfl — APPROVED** [review]
Looks good to me!

File: docs/source/data.rst (hunk 2 above)
Comment (znicholls) [reply docs/source/data.rst:11]: Nice

**Pull request merged**